docs/on-prem.rst

```diff
@@ -6,14 +6,15 @@ Module 1: On-Prem Tutorial
 .. _cp-demo-run:
 
 ====================
-Run on-prem cluster
+Run on-prem Confluent Platform cluster
```

Comment:
```suggestion
Run on-prem |cp| cluster
```

Reply:
Please check to make sure that the heading underline matches the length of the text in source (_after_ the edit to use the `|cp|` substitution above. If the underline is too short, it will cause a docs build break.